templates/html_report/composer.php

```diff
@@ -0,0 +1,100 @@
+<?php
+require __DIR__ . '/_header.php'; ?>
+
+<?php
+$packages = isset($project['composer']['packages']) ? $project['composer']['packages'] : [];
+usort($packages, function ($a, $b) {
+    return strcmp($a->name, $b->name);
+});
+if (empty($packages)) {
+    echo '<div class="row"><div class="column"><div class="bloc">No composer.json file found in this project</div></div></div>';
+    require __DIR__ . '/_footer.php';
+    return;
+}
+?>
+
+
+<div class="row">
+    <div class="column">
+        <div class="bloc">
+            <h4><?php echo sizeof($packages); ?> Composer dependencies</h4>
+
+            <div class="list">
+                <?php foreach ($packages as $package) { ?>
+                    <div class="list-item">
+                        <div class="list-item-title">
+                            <a target="_blank" href="https://packagist.org/packages/<?php echo $package->name; ?>">
+                                <?php echo $package->name; ?>
+                            </a>
+                        </div>
+                        <div class="help">
+                            <div class="help-inner">
+                                <?php if ('outdated' === $package->status) { ?>
+                                    <span class="help-warning" style="position: absolute; right: 10px;">This package should be updated.</span>
+                                <?php } ?>
+                                <span class="badge"><?php echo $package->type; ?></span>
+                                <?php echo $package->description; ?>
+                            </div>
+                        </div>
+
+                        <div class="list-item-content">
+                            <?php $width = sprintf('%d%%', round(100 / 9, 0)); ?>
+                            <table class="table-metrics" style="=margin-left:50px;">
```

Comment:
```suggestion
                            <table class="table-metrics" style="margin-left:50px;">
```

Probably a typo here.